ote/src/clj/ote/services/transit_visualization.sql

```diff
@@ -170,3 +170,22 @@ SELECT p.id, p.created,
   JOIN "external-interface-description" eid ON p."external-interface-description-id" = eid.id
  WHERE p."transport-service-id" = :service-id AND p."deleted?" = FALSE
  ORDER BY p.created DESC;
+
+-- name: detected-changes-for-date
```

Comment:
Better name for the query: detected-changed-to-service-by-date

Reply:
Roger